ctapipe/irf/irf_classes.py

```diff
@@ -115,6 +116,7 @@ class EventPreSelector(Component):
     )
 
     def _preselect_events(self, events):
+        tc = self.parent.tc
```

Comment:
Why is here an explicit usage of the parent needed? We use this nowhere else and it prevents standalone usage (and testing) of this component.